src/compiler/emitter.ts

```diff
@@ -2138,6 +2138,72 @@ module ts {
             function getTemplateLiteralAsStringLiteral(node: LiteralExpression): string {
                 return '"' + escapeString(node.text) + '"';
             }
+            
+            function emitDownlevelRawTemplateLiteral(node: LiteralExpression) {
+                var text = getSourceTextOfNodeFromSourceFile(currentSourceFile, node);

```

Comment:
I'm torn.  On one hand, this brings us closer to what the user already wrote.  ON the other, it means you need to do a lot more processing.  Why not just use node.text instead?  It will already have stripped off the } and ${  bits, and will already have changes newlines appropriately.